pytaxonkit.py

```diff
@@ -1112,10 +1113,17 @@ def lca(
         else:
             return None
     results = []
+    query_str_list = []
     for line in out.strip().split("\n"):
         queries, lcataxid = line.split("\t")
+        query_str_list.append(queries)
         results.append(int(lcataxid))
+    missing_ids = set(id_str_list) - set(query_str_list)
+    missing_indexes = [id_str_list.index(missing_id) for missing_id in missing_ids]
+    for missing_index in sorted(missing_indexes):
+        results.insert(missing_index, 0)
```

Comment:
To remedy this, we determine which taxids have been skipped over by doing some set calculations. This is done using the `id_str_list` created when handling the input to `pytaxonkit.lca` and the results from `taxonkit.lca`. We then find the index of missing queries in the original input list. We sort these indexes and insert a `0` into the results at the given index.